components/TextFields/tests/unit/ContainedInputView/MDCBaseTextFieldTests.m

```diff
@@ -32,71 +34,457 @@ - (UIView *)createSideView {
 }
 
 - (void)testLeadingViewLTR {
```

Comment:
For each of these tests please consider describing what the test is testing to help with readability.

E.g. `testLeadingViewEqualsLeftViewInLTR`

Reply:
Done